docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md

```diff
@@ -281,15 +281,14 @@ Completeness checklist for ExpectColumnMaxToBeBetweenCustom:
 
 ### 8. Linting
 
-Finally, we need to lint our now-functioning Custom Expectation. Our CI system will test your code using `black`, `isort`, `flake8`, and `pyupgrade`. 
+Finally, we need to lint our now-functioning Custom Expectation. Our CI system will test your code using `black`, `isort`, and `ruff`.
```

Comment:
By definition, isn't a linter is something that just does static analysis but doesn't fix the issue for you? This let's us run various linting tools in parallel?

Should we really be using `--fix` in all cases? If so, would we want to add the same command for `black` and `isort` below?

Reply:
okay I'm fine with this